ecs-cli/modules/cli/compose/project/project_parseV1V2_test.go

```diff
@@ -0,0 +1,344 @@
+
+	assert.Equal(t, command, web.Command, "Expected Command to match")
+	assert.Equal(t, cpuShares, web.CPU, "Expected CPU to match")
+	assert.Equal(t, dnsSearchDomains, web.DNSSearchDomains, "Expected DNSSearchDomains to match")
+	assert.Equal(t, dnsServers, web.DNSServers, "Expected DNSServers to match")
+	assert.Equal(t, labels, web.DockerLabels, "Expected docker labels to match")
+	assert.Equal(t, securityOpts, web.DockerSecurityOptions, "Expected DockerSecurityOptions to match")
+	assert.Equal(t, entryPoint, web.Entrypoint, "Expected EntryPoint to be match")
+	assert.Equal(t, env, web.Environment, "Expected Environment to match")
+	assert.Equal(t, extraHosts, web.ExtraHosts, "Expected ExtraHosts to match")
+	assert.Equal(t, hostname, web.Hostname, "Expected Hostname to match")
+	assert.Equal(t, webImage, web.Image, "Expected Image to match")
+	assert.Equal(t, links, web.Links, "Expected Links to match")
+	assert.Equal(t, logDriver, web.LogConfiguration.LogDriver, "Expected LogDriver to match")
+	assert.Equal(t, logOpts, web.LogConfiguration.Options, "Expected LogConfiguration options to match")
+	assert.Equal(t, memory, web.Memory, "Expected memory to match")
+	assert.Equal(t, mountPoints, web.MountPoints, "Expected MountPoints to match")
+	assert.Equal(t, ports, web.PortMappings, "Expected PortMappings to match")
+	assert.Equal(t, privileged, web.Privileged, "Expected Privileged to match")
+	assert.Equal(t, readonly, web.ReadOnly, "Expected ReadOnly to match")
+	assert.Equal(t, shmSize, web.ShmSize, "Expected ShmSize to match")
+	assert.Equal(t, user, web.User, "Expected user to match")
+	assert.Equal(t, workingDir, web.WorkingDirectory, "Expected WorkingDirectory to match")
+}
+
+func TestParseV1V2_Version2Files(t *testing.T) {
+	// expected ContainerConfig values
+	wordpressImage := "wordpress"
+	mysqlImage := "mysql"
+
+	capAdd := []string{"ALL"}
+	capDrop := []string{"NET_ADMIN", "SYS_ADMIN"}
+	ports := []*ecs.PortMapping{
+		{
+			ContainerPort: aws.Int64(80),
+			HostPort:      aws.Int64(80),
+			Protocol:      aws.String("tcp"),
+		},
+	}
+	memoryReservation := int64(476) // 500000000 / miB
+	memory := int64(512)
+	mountPoints := []*ecs.MountPoint{
+		{
+			ContainerPath: aws.String("/tmp/cache"),
+			ReadOnly:      aws.Bool(false),
+			SourceVolume:  aws.String("banana"),
+		},
+		{
+			ContainerPath: aws.String("/tmp/cache"),
+			ReadOnly:      aws.Bool(false),
+			SourceVolume:  aws.String("volume-1"),
+		},
+		{
+			ContainerPath: aws.String("/tmp/cache"),
+			ReadOnly:      aws.Bool(true),
+			SourceVolume:  aws.String("volume-2"),
+		},
+	}
+	shmSize := int64(1024) // 1 gb = 1024 miB
+	tmpfs := []*ecs.Tmpfs{
+		{
+			ContainerPath: aws.String("/run"),
+			MountOptions:  aws.StringSlice([]string{}),
+			Size:          aws.Int64(1024),
+		},
+		{
+			ContainerPath: aws.String("/tmp"),
+			MountOptions:  aws.StringSlice([]string{"ro", "rw"}),
+			Size:          aws.Int64(64),
+		},
+	}
+
+	composeV2FileString := `version: '2'
```

Comment:
similar to above: can we add an arg and check for a `volumes_from` value?
https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes_from

Reply:
done.